cfgov/unprocessed/js/organisms/FilterableListControls.js

```diff
@@ -72,7 +77,52 @@ function FilterableListControls( element ) {
    * Initialize FilterableListControls events.
    */
   function _initEvents() {
-    _form.addEventListener( 'submit', _formSubmitted );
+    var labelDom = _dom.querySelector( '.o-expandable_label' );
+    var label;
+
+    if ( labelDom ) {
+      label = labelDom.textContent.trim();
+    }
+
+    _expandable.addEventListener( 'expandBegin',
+    function sendEvent( ) {
+      Analytics.sendEvent( 'Filter:open', label );
+    } );
+
+    _expandable.addEventListener( 'collapseBegin',
+    function sendEvent( ) {
+      Analytics.sendEvent( 'Filter:close', label );
+    } );
+
+    _form.addEventListener( 'change', function sendEvent( event ) {
+      var action;
+      var field = event.target;
+      var fieldValue;
+      var fieldLabel;
+
+      if ( !field ) {
+        return;
+      } else if ( !_cachedLabels[field.name] ) {
+        _cachedLabels[field.name] = {
+          label:  _getLabelText( field )
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2308/commits/c02e2cc8406b575f3fef465b8b2f4ac920e47259